.github/workflows/kotlin-sample-ci.yml

```diff
@@ -6,10 +6,14 @@ on:
   pull_request:
```

Comment:
Do we also need to trigger on `pull_request_target` event for Kotlin Multiplatform CI?

Reply:
as it's the sample that's fine, it doesn't use the big runner